node-fs-writefile/cp.js

```diff
@@ -0,0 +1,10 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+const argv = process.argv.slice(2);
```

Comment:
It would be nice to explicitly extract the src and dest filenames instead of having to access them with array indices.